corehq/pillows/synclog.py

```diff
@@ -23,6 +23,7 @@ def get_user_sync_history_pillow(pillow_id='UpdateUserSyncHistoryPillow', **kwar
     """
     This gets a pillow which iterates through all synclogs
     """
+    #Todo; convert to SQL?
```

Comment:
yes

Reply:
Cool. Is there an example of SQL pillow processor?

Reply:
the 'processor' won't change much. The big difference is that the pillow will need to use a `KafkaChangeFeed` instead of a `CouchChangeFeed`. Here's an example: https://github.com/dimagi/commcare-hq/blob/3e88dbe4554316981e7f9a43fde34aa8870702c9/corehq/pillows/app_submission_tracker.py#L24

Reply:
@sravfeyn is this going to be included in this PR?